storage/postgres/migrations/plugin/20250721200403_delete_policy_constraints.sql

```diff
@@ -0,0 +1,75 @@
+-- +goose Up
```

Comment:
The triggers for policy deletion should go into a system migration on verifier - otherwise every plugin will need to reimplement these.